js/profile-geonovum.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
+// In case everything else fails, we want the error
+window.addEventListener("error", ev => {
+  console.error(ev.error, ev.message, ev);
+});
+
+// this is only set in a build, not at all in the dev environment
+require.config({
+  paths: {
+    clipboard: "deps/clipboard",
+    hyperhtml: "deps/hyperhtml",
+    idb: "deps/idb",
+    jquery: "deps/jquery",
+    marked: "deps/marked",
+    pluralize: "deps/pluralize",
+    text: "deps/text",
+    webidl2: "deps/webidl2",
+  },
+  shim: {
+    shortcut: {
+      exports: "shortcut",
+    },
+    highlight: {
+      exports: "hljs",
+    },
+  },
+});
+
+define([
+  // order is significant
+  "./core/base-runner",
+  "./core/ui",
+  "./core/reindent",
+  "./core/location-hash",
+  "./core/l10n",
+  "./w3c/defaults",
```

Comment:
We probably don't want this one... we should make a geonovum/defaults.js instead.

Reply:
We have a geonovum/defaults.js already. For now, to make each PR small, should I just remove the ./w3c/defaults? I could also try to replace it directly with the geonovum defaults, but a following PR will be a bit more complicated if we do this. But it makes sense to set it up correctly first. Also with the styles etc

Reply:
Yeah, let's remove it for now. 
```suggestion

```